js/src/css/shared/_woocommerce-admin.scss

```diff
@@ -10,12 +10,11 @@
 	.woocommerce-layout {
 		padding-top: 0;
 
-		// set the layout header width to 0 to hide it.
 		// do not set the header to display: none,
 		// because we want the offsetHeight value
 		// and set it to wpbody-content margin-top to counter back.
 		.woocommerce-layout__header {
-			width: 0;
+			visibility: hidden;
```

Comment:
💅 
I think it's fine to use `display: none;` since the original use in `useWPBodyMarginOffsetEffect` (a75ed9f8ed825af7f880d1db005a5dfb45beac8d and ddcaecd96657f835c36750b30951cdb193e34761) had been replaced with another approach in #1206.

Reply:
Thanks for the review and the suggestion @eason9487 . Since visibility:hidden causes no damage I prefer to maintain the code as closest as before. But totally agree to have a further PR with display:none approach if needed.

Reply:
I happened to notice the code comment of this CSS style was outdated. Keeping `visibility: hidden;` is ok as well but the outdated code comment might be confusing.